docs/About/index.md

```diff
@@ -0,0 +1,23 @@
+# About 
+
+GTFS.org is the central documentation platform for the General Transit Feed Specification. This site and its contents are maintained by [MobilityData](https://mobilitydata.org/).
+
+Thank you to [Andrew Byrd](https://www.linkedin.com/in/byrdandrew) for purchasing and lending use of the GTFS.org domain name and providing leadership for the community.
+
+## Contributing to GTFS.org
+
+To propose edits to the content of this website, contact MobilityData at [documentation@mobilitydata.org](mailto:documentation@mobilitydata.org) or visit the [GTFS.org GitHub Repository](https://github.com/mobilitydata/gtfs.org) to submit your edits as pull requests.
+
+## GTFS evolution
+
+GTFS started with a collaboration between TriMet in Portland, Oregon, and Google. TriMet worked with Google to format their transit data into an easily maintainable and consumable format that could be imported into Google Maps. This transit data format was originally known as the Google Transit Feed Specification (GTFS).
+
+As a result of developer innovation, GTFS data is now being used by a variety of third-party software applications for many different purposes, including trip planning, timetable creation, mobile data, data visualization, accessibility, analysis tools for planning, and real-time information systems. In 2010, the GTFS format name was changed to the General Transit Feed Specification to accurately represent its use in many different applications outside of Google products.
```

Comment:
Add:
- in 2011, GTFS Realtime was created
- in 2019, the non-profit [MobilityData](https://mobilitydata.org/) was established to further maintain GTFS with the help of the community